src/openms/source/FORMAT/MSPFile.cpp

```diff
@@ -98,26 +95,9 @@ namespace OpenMS
     String line;
     ifstream is(filename.c_str());
 
-    Map<String, double> mod_to_mass;
-    mod_to_mass["Oxidation"] = 15.994915;
-    mod_to_mass["Carbamidomethyl"] = 57.02146;
-    mod_to_mass["ICAT_light"] = 227.126991;
-    mod_to_mass["ICAT_heavy"] = 236.157185;
-    mod_to_mass["AB_old_ICATd0"] = 442.224991;
-    mod_to_mass["AB_old_ICATd8"] = 450.275205;
-    mod_to_mass["Acetyl"] = 42.0106;
```

Comment:
what about those?

Reply:
apparently not needed

Reply:
not needed in the test files or in general?

Reply:
the variable was not used anymore in the file. Not sure about other files, but I think compilation would have failed.